packages/components/psammead-program-card/src/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import styled from 'styled-components';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import Paragraph from '@bbc/psammead-paragraph';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  C_EBON,
+  C_METAL,
+  C_SHADOW,
+  C_WHITE,
+  C_POSTBOX,
+  C_KINGFISHER,
+} from '@bbc/psammead-styles/colours';
+import {
+  getSansRegular,
+  getSerifRegular,
+  getSansBold,
+} from '@bbc/psammead-styles/font-styles';
+import {
+  getLongPrimer,
+  getMinion,
+  getPica,
+} from '@bbc/gel-foundations/typography';
+import { Link } from '@bbc/psammead-story-promo';
+
+const programStateColors = {
+  live: {
+    backgroundColor: C_POSTBOX,
+    headerLabelColor: C_POSTBOX,
+    headerTextColor: C_EBON,
+    durationColor: C_WHITE,
+  },
+  next: {
+    backgroundColor: C_WHITE,
+    headerLabelColor: C_KINGFISHER,
+    headerTextColor: C_METAL,
+    durationColor: C_KINGFISHER,
+  },
+  onDemand: {
+    backgroundColor: C_EBON,
+    headerLabelColor: '',
+    headerTextColor: C_EBON,
+    durationColor: C_WHITE,
+  },
+};
+
+const CardWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+  background-color: ${C_WHITE};
+  position: relative;
+`;
+
+const TextWrapper = styled.div`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const HeadingWrapper = styled.h3`
+  ${({ service }) => service && getSerifRegular(service)};
+  ${({ script }) => script && getPica(script)};
+  color: ${({ headerTextColor }) => headerTextColor};
+  margin: 0; /* Reset */
+`;
+
+const LabelWrapper = styled.span`
+  ${({ service }) => service && getSansBold(service)};
+  ${({ script }) => script && getPica(script)};
+  text-transform: uppercase;
+  color: ${({ headerLabelColor }) => headerLabelColor};
+`;
+
+const DateWrapper = styled.div`
+  color: ${C_SHADOW};
+  padding: ${GEL_SPACING} 0;
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getPica(script)};
+`;
+
+const SummaryWrapper = styled(Paragraph)`
```

Comment:
If we're overriding all of the styles, should we just be using a `p` tag here rather than overriding the styles from the `Paragraph` component? The bulletin also uses a paragraph with different styling from the `Paragraph` component.